hydra-main/src/main/java/com/addthis/hydra/job/spawn/balancer/HostScore.java

```diff
@@ -16,11 +16,13 @@
 final class HostScore {
 
     private final double meanActiveTasks;
+    private final long diskFreeBytes;
     private final double usedDiskPercent;
     private final double overallScore;
 
-    HostScore(double meanActiveTasks, double usedDiskPercent, double overallScore) {
+    HostScore(double meanActiveTasks, double usedDiskPercent, long diskFreeBytes, double overallScore) {
         this.meanActiveTasks = meanActiveTasks;
+        this.diskFreeBytes = diskFreeBytes;
```

Comment:
@tea-dragon - you are right. I will remove this and just use the host states that are in scope. Thanks for the feedback!